fossevents/events/views.py

```diff
@@ -14,3 +18,18 @@ def event_detail(request, pk, slug=None, template_name='events/event_detail.html
         'event': event,
     }
     return render(request, template_name, ctx)
+
+
+class EventCreateView(SendMailPostSaveMixin, CreateView):
+    form_class = EventForm
+    template_name = 'events/event_form.html'
+
+
+class EventUpdateView(SendMailPostSaveMixin, UpdateView):
```

Comment:
I am working on checking if something is updated and if we can get the diff. I'll merge it in next PR.